packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1013,5 +1013,5 @@ authfactor.challenge.soft_token.invalid_passcode = Your code doesn't match our r
 authfactor.challenge.soft_token.used_passcode = Each code can only be used once. Please wait for a new code and try again.
 
 # rate limit
-oie.tooManyRequests = Too many requests
+oie.tooManyRequests = Too many attempts. Try again later.
```

Comment:
This will potentially break selenium tests in okta-core. Please create a downstream branch and run tests on it - followed by a PR to fix the tests before merging this. Otherwise we will be putting the monthly release to jeopardy.

Reply:
Also left another comment regarding the message itself.

Reply:
sure. I will run selenium test before merging.